### ISSUE TITLE
[Test Failure] Individual TAO Test Failure in prtest/win/vsi/p1 on PR #634 (Venus)

### ISSUE THREAD

**Issue by jmarolf** [dotnet/roslyn#9557]
PR: [#634](https://github.com/dotnet/roslyn-internal/pull/634) *Only log formatting events in telemetry when they are enacted by the …* by @jmarolf
Failure: http://dotnet-ci.cloudapp.net/job/roslyn-internal_prtest_win_vsi_p1/173/

**1 Test Failures:**
BasicVenus.Generate Type Into App_Code Folder

**Issue 1: Individual TAO Test Failure**
Failing integration tests: 

BasicVenus.xml

```
02:31:00 ================================================================================
02:31:00 Dumping logs of all failed TAO tests.
02:31:00 ================================================================================
02:31:00 ================================================================================
02:31:00  Dumping log file d:\j\workspace\roslyn-intern---eef4959b\Open\Binaries\Release\VSIntegrationTestLogs\BasicVenus.txt
02:31:00 Test: BasicVenus started: 3/8/2016 9:39:28 AM
02:31:00 	Executing action [3/8/2016 9:39:28 AM] StartTarget
02:31:00 Begin Waiting for Apex Communication
02:31:00 	Executing action [3/8/2016 9:39:28 AM] SetToolsOption
02:31:00 Previous option value: TextEditor > HTML > InsertTabs = False)
02:31:00 Setting option: TextEditor > HTML > InsertTabs = false)
02:31:00 	Executing action [3/8/2016 9:39:28 AM] SetFeatureOption
02:31:00 	Executing action [3/8/2016 9:39:28 AM] AddProject
02:31:00 	Executing action [3/8/2016 9:39:29 AM] AddItem
02:31:00  Scenario: Tab completing an event inserts Event Handler
02:31:00 	Executing action [3/8/2016 9:39:30 AM] OpenFile
02:31:00 DEBUG: executed DTE command File.OpenFile.
02:31:00 DEBUG: start polling for code window
02:31:00 DEBUG: polling complete
02:31:00 	Executing action [3/8/2016 9:39:30 AM] PlaceCursor
02:31:00 	Executing action [3/8/2016 9:39:30 AM] InsertText
02:31:00 	Executing action [3/8/2016 9:39:30 AM] PlaceCursor
02:31:00 	Executing action [3/8/2016 9:39:30 AM] Venus
02:31:00 	Executing action [3/8/2016 9:39:34 AM] WaitForWorkspace
02:31:00 	Executing action [3/8/2016 9:39:34 AM] SendKeys
02:31:00 	Executing action [3/8/2016 9:39:36 AM] OpenFile
02:31:00 DEBUG: executed DTE command File.OpenFile.
02:31:00 DEBUG: start polling for code window
02:31:00 DEBUG: polling complete
02:31:00 	Executing action [3/8/2016 9:39:36 AM] VerifyEditorContainsText
02:31:00 	Executing action [3/8/2016 9:39:36 AM] VerifyCaretPosition
02:31:00 	Executing action [3/8/2016 9:39:36 AM] CloseFile
02:31:00 	Executing action [3/8/2016 9:39:36 AM] CloseFile
02:31:00 	Executing action [3/8/2016 9:39:36 AM] CloseFile
02:31:00  Scenario: Double clicking a button generates corresponding event handler
02:31:00 	Executing action [3/8/2016 9:39:36 AM] OpenFile
02:31:00 DEBUG: executed DTE command File.OpenFile.
02:31:00 DEBUG: start polling for code window
02:31:00 DEBUG: polling complete
02:31:00 	Executing action [3/8/2016 9:39:36 AM] PlaceCursor
02:31:00 	Executing action [3/8/2016 9:39:36 AM] InsertText
02:31:00 	Executing action [3/8/2016 9:39:36 AM] PlaceCursor
02:31:00 	Executing action [3/8/2016 9:39:36 AM] Venus
02:31:00 	Executing action [3/8/2016 9:39:38 AM] WaitForWorkspace
02:31:00 	Executing action [3/8/2016 9:39:38 AM] ExecuteCommand
02:31:00 	Executing action [3/8/2016 9:39:38 AM] VerifyEditorContainsText
02:31:00 	Executing action [3/8/2016 9:39:38 AM] VerifyCaretPosition
02:31:00 	Executing action [3/8/2016 9:39:38 AM] CloseFile
02:31:00 	Executing action [3/8/2016 9:39:38 AM] CloseFile
02:31:00 	Executing action [3/8/2016 9:39:38 AM] CloseFile
02:31:00  Scenario: Code Actions are disabled across projections
02:31:00 	Executing action [3/8/2016 9:39:38 AM] OpenFile
02:31:00 DEBUG: executed DTE command File.OpenFile.
02:31:00 DEBUG: start polling for code window
02:31:00 DEBUG: polling complete
02:31:00 	Executing action [3/8/2016 9:39:38 AM] PlaceCursor
02:31:00 	Executing action [3/8/2016 9:39:38 AM] InsertText
02:31:00 	Executing action [3/8/2016 9:39:38 AM] Venus
02:31:00 	Executing action [3/8/2016 9:39:38 AM] PlaceCursor
02:31:00 	Executing action [3/8/2016 9:39:38 AM] VerifyCodeActions
02:31:00 	Executing action [3/8/2016 9:40:04 AM] CloseFile
02:31:00 	Executing action [3/8/2016 9:40:04 AM] CloseFile
02:31:00  Scenario: Code Actions are disabled within projections
02:31:00 	Executing action [3/8/2016 9:40:04 AM] OpenFile
02:31:00 DEBUG: executed DTE command File.OpenFile.
02:31:00 DEBUG: start polling for code window
02:31:00 DEBUG: polling complete
02:31:00 	Executing action [3/8/2016 9:40:04 AM] PlaceCursor
02:31:00 	Executing action [3/8/2016 9:40:04 AM] InsertText
02:31:00 	Executing action [3/8/2016 9:40:04 AM] Venus
02:31:00 	Executing action [3/8/2016 9:40:05 AM] SelectText
02:31:00 	Executing action [3/8/2016 9:40:05 AM] VerifyCodeActions
02:31:00 	Executing action [3/8/2016 9:40:30 AM] CloseFile
02:31:00 	Executing action [3/8/2016 9:40:30 AM] CloseFile
02:31:00  Scenario: Code Generation features are disabled within projections
02:31:00 	Executing action [3/8/2016 9:40:30 AM] OpenFile
02:31:00 DEBUG: executed DTE command File.OpenFile.
02:31:00 DEBUG: start polling for code window
02:31:00 DEBUG: polling complete
02:31:00 	Executing action [3/8/2016 9:40:30 AM] PlaceCursor
02:31:00 	Executing action [3/8/2016 9:40:30 AM] InsertText
02:31:00 	Executing action [3/8/2016 9:40:30 AM] PlaceCursor
02:31:00 	Executing action [3/8/2016 9:40:30 AM] InsertText
02:31:00 	Executing action [3/8/2016 9:40:30 AM] Venus
02:31:00 	Executing action [3/8/2016 9:40:31 AM] PlaceCursor
02:31:00 	Executing action [3/8/2016 9:40:31 AM] VerifyCodeActions
02:31:00 	Executing action [3/8/2016 9:40:56 AM] CloseFile
02:31:00 	Executing action [3/8/2016 9:40:56 AM] CloseFile
02:31:00  Scenario: Script Block: Format Document command
02:31:00 	Executing action [3/8/2016 9:40:56 AM] OpenFile
02:31:00 DEBUG: executed DTE command File.OpenFile.
02:31:00 DEBUG: start polling for code window
02:31:00 DEBUG: polling complete
02:31:00 	Executing action [3/8/2016 9:40:56 AM] PlaceCursor
02:31:00 	Executing action [3/8/2016 9:40:56 AM] InsertText
02:31:00 	Executing action [3/8/2016 9:40:56 AM] WaitForWorkspace
02:31:00 	Executing action [3/8/2016 9:40:57 AM] Venus
02:31:00 	Executing action [3/8/2016 9:40:57 AM] FormatDocument
02:31:00 	Executing action [3/8/2016 9:40:57 AM] VerifyEditorContainsText
02:31:00 	Executing action [3/8/2016 9:40:57 AM] CloseFile
02:31:00 	Executing action [3/8/2016 9:40:57 AM] CloseFile
02:31:00  Scenario: Script Block: Smart Indent On Enter
02:31:00 	Executing action [3/8/2016 9:40:57 AM] OpenFile
02:31:00 DEBUG: executed DTE command File.OpenFile.
02:31:00 DEBUG: start polling for code window
02:31:00 DEBUG: polling complete
02:31:00 	Executing action [3/8/2016 9:40:57 AM] PlaceCursor
02:31:00 	Executing action [3/8/2016 9:40:57 AM] InsertText
02:31:00 	Executing action [3/8/2016 9:40:57 AM] Venus
02:31:00 	Executing action [3/8/2016 9:40:57 AM] PlaceCursor
02:31:00 	Executing action [3/8/2016 9:40:57 AM] ExecuteCommand
02:31:00 	Executing action [3/8/2016 9:40:57 AM] VerifyCaretPosition
02:31:00 	Executing action [3/8/2016 9:40:57 AM] CloseFile
02:31:00 	Executing action [3/8/2016 9:40:57 AM] CloseFile
02:31:00  Scenario: Script Block: Smart Indent On Moving out of current line
02:31:00 	Executing action [3/8/2016 9:40:57 AM] OpenFile
02:31:00 DEBUG: executed DTE command File.OpenFile.
02:31:00 DEBUG: start polling for code window
02:31:00 DEBUG: polling complete
02:31:00 	Executing action [3/8/2016 9:40:57 AM] PlaceCursor
02:31:00 	Executing action [3/8/2016 9:40:57 AM] InsertText
02:31:00 	Executing action [3/8/2016 9:40:57 AM] WaitForWorkspace
02:31:00 	Executing action [3/8/2016 9:40:58 AM] PlaceCursor
02:31:00 	Executing action [3/8/2016 9:40:58 AM] SendKeys
02:31:00 	Executing action [3/8/2016 9:40:58 AM] ExecuteCommand
02:31:00 	Executing action [3/8/2016 9:40:58 AM] VerifyEditorContainsText
02:31:00 	Executing action [3/8/2016 9:40:58 AM] CloseFile
02:31:00 	Executing action [3/8/2016 9:40:58 AM] CloseFile
02:31:00  Scenario: Script Block: Commit and Smart Indent On Moving out of current line
02:31:00 	Executing action [3/8/2016 9:40:58 AM] OpenFile
02:31:00 DEBUG: executed DTE command File.OpenFile.
02:31:00 DEBUG: start polling for code window
02:31:00 DEBUG: polling complete
02:31:00 	Executing action [3/8/2016 9:40:58 AM] PlaceCursor
02:31:00 	Executing action [3/8/2016 9:40:58 AM] InsertText
02:31:00 	Executing action [3/8/2016 9:40:58 AM] WaitForWorkspace
02:31:00 	Executing action [3/8/2016 9:40:59 AM] PlaceCursor
02:31:00 	Executing action [3/8/2016 9:40:59 AM] SendKeys
02:31:00 	Executing action [3/8/2016 9:40:59 AM] ExecuteCommand
02:31:00 	Executing action [3/8/2016 9:40:59 AM] VerifyEditorContainsText
02:31:00 	Executing action [3/8/2016 9:40:59 AM] CloseFile
02:31:00 	Executing action [3/8/2016 9:40:59 AM] CloseFile
02:31:00  Scenario: Nugget: Format Document command
02:31:00 	Executing action [3/8/2016 9:40:59 AM] OpenFile
02:31:00 DEBUG: executed DTE command File.OpenFile.
02:31:00 DEBUG: start polling for code window
02:31:00 DEBUG: polling complete
02:31:00 	Executing action [3/8/2016 9:40:59 AM] PlaceCursor
02:31:00 	Executing action [3/8/2016 9:40:59 AM] InsertText
02:31:00 	Executing action [3/8/2016 9:40:59 AM] WaitForWorkspace
02:31:00 	Executing action [3/8/2016 9:40:59 AM] FormatDocument
02:31:00 	Executing action [3/8/2016 9:40:59 AM] VerifyEditorContainsText
02:31:00 	Executing action [3/8/2016 9:40:59 AM] CloseFile
02:31:00 	Executing action [3/8/2016 9:40:59 AM] CloseFile
02:31:00  Scenario: Nugget: Smart Indent on Enter
02:31:00 	Executing action [3/8/2016 9:40:59 AM] OpenFile
02:31:00 DEBUG: executed DTE command File.OpenFile.
02:31:00 DEBUG: start polling for code window
02:31:00 DEBUG: polling complete
02:31:00 	Executing action [3/8/2016 9:41:00 AM] PlaceCursor
02:31:00 	Executing action [3/8/2016 9:41:00 AM] InsertText
02:31:00 	Executing action [3/8/2016 9:41:00 AM] PlaceCursor
02:31:00 	Executing action [3/8/2016 9:41:00 AM] ExecuteCommand
02:31:00 	Executing action [3/8/2016 9:41:00 AM] VerifyCaretPosition
02:31:00 	Executing action [3/8/2016 9:41:00 AM] InsertText
02:31:00 	Executing action [3/8/2016 9:41:00 AM] ExecuteCommand
02:31:00 	Executing action [3/8/2016 9:41:00 AM] VerifyCaretPosition
02:31:00 	Executing action [3/8/2016 9:41:00 AM] CloseFile
02:31:00 	Executing action [3/8/2016 9:41:00 AM] CloseFile
02:31:00  Scenario: Error list shows correctly mapped line numbers and column information
02:31:00 	Executing action [3/8/2016 9:41:00 AM] OpenFile
02:31:00 DEBUG: executed DTE command File.OpenFile.
02:31:00 DEBUG: start polling for code window
02:31:00 DEBUG: polling complete
02:31:00 	Executing action [3/8/2016 9:41:00 AM] PlaceCursor
02:31:00 	Executing action [3/8/2016 9:41:00 AM] InsertText
02:31:00 	Executing action [3/8/2016 9:41:00 AM] Venus
02:31:00 	Executing action [3/8/2016 9:41:00 AM] ExecuteCommand
02:31:00 	Executing action [3/8/2016 9:41:00 AM] OpenFile
02:31:00 DEBUG: executed DTE command File.OpenFile.
02:31:00 DEBUG: start polling for code window
02:31:00 DEBUG: polling complete
02:31:00 	Executing action [3/8/2016 9:41:00 AM] VerifyErrorList
02:31:00 	Executing action [3/8/2016 9:41:17 AM] Venus
02:31:00 	Executing action [3/8/2016 9:41:17 AM] VerifyErrorList
02:31:00 	Executing action [3/8/2016 9:41:25 AM] Venus
02:31:00 	Executing action [3/8/2016 9:41:25 AM] VerifyCaretPosition
02:31:00 	Executing action [3/8/2016 9:41:25 AM] CloseFile
02:31:00 	Executing action [3/8/2016 9:41:25 AM] CloseFile
02:31:00  Scenario: Error list includes out of proc build errors
02:31:00 	Executing action [3/8/2016 9:41:25 AM] OpenFile
02:31:00 DEBUG: executed DTE command File.OpenFile.
02:31:00 DEBUG: start polling for code window
02:31:00 DEBUG: polling complete
02:31:00 	Executing action [3/8/2016 9:41:25 AM] OpenFile
02:31:00 DEBUG: executed DTE command File.OpenFile.
02:31:00 DEBUG: start polling for code window
02:31:00 DEBUG: polling complete
02:31:00 	Executing action [3/8/2016 9:41:25 AM] SetEditorText
02:31:00 	Executing action [3/8/2016 9:41:26 AM] Venus
02:31:00 	Executing action [3/8/2016 9:41:26 AM] Build
02:31:00 	Executing action [3/8/2016 9:41:26 AM] Venus
02:31:00 	Executing action [3/8/2016 9:41:26 AM] ExecuteCommand
02:31:00 	Executing action [3/8/2016 9:41:26 AM] VerifyErrorList
02:31:00 	Executing action [3/8/2016 9:41:42 AM] Build
02:31:00 	Executing action [3/8/2016 9:41:42 AM] VerifyErrorList
02:31:00 	Executing action [3/8/2016 9:41:51 AM] CloseFile
02:31:00 	Executing action [3/8/2016 9:41:51 AM] CloseFile
02:31:00  Scenario: Snippets in Nuggets
02:31:00 	Executing action [3/8/2016 9:41:51 AM] OpenFile
02:31:00 DEBUG: executed DTE command File.OpenFile.
02:31:00 DEBUG: start polling for code window
02:31:00 DEBUG: polling complete
02:31:00 	Executing action [3/8/2016 9:41:51 AM] PlaceCursor
02:31:00 	Executing action [3/8/2016 9:41:51 AM] InsertText
02:31:00 	Executing action [3/8/2016 9:41:51 AM] PlaceCursor
02:31:00 	Executing action [3/8/2016 9:41:51 AM] Venus
02:31:00 	Executing action [3/8/2016 9:41:51 AM] ExecuteCommand
02:31:00 	Executing action [3/8/2016 9:41:51 AM] VerifyEditorContainsText
02:31:00 	Executing action [3/8/2016 9:41:51 AM] VerifySnippets
02:31:00 	Executing action [3/8/2016 9:41:51 AM] CloseFile
02:31:00 	Executing action [3/8/2016 9:41:51 AM] CloseFile
02:31:00  Scenario: Snippets in Nuggets: from snippet picker
02:31:00 	Executing action [3/8/2016 9:41:51 AM] OpenFile
02:31:00 DEBUG: executed DTE command File.OpenFile.
02:31:00 DEBUG: start polling for code window
02:31:00 DEBUG: polling complete
02:31:00 	Executing action [3/8/2016 9:41:51 AM] PlaceCursor
02:31:00 	Executing action [3/8/2016 9:41:51 AM] InsertText
02:31:00 	Executing action [3/8/2016 9:41:51 AM] PlaceCursor
02:31:00 	Executing action [3/8/2016 9:41:51 AM] Venus
02:31:00 	Executing action [3/8/2016 9:41:51 AM] VerifySnippets
02:31:00 	Executing action [3/8/2016 9:41:51 AM] VerifyEditorContainsText
02:31:00 	Executing action [3/8/2016 9:41:51 AM] VerifySnippets
02:31:00 	Executing action [3/8/2016 9:41:51 AM] CloseFile
02:31:00 	Executing action [3/8/2016 9:41:51 AM] CloseFile
02:31:00  Scenario: Snippets insert aspx style Imports statements
02:31:00 	Executing action [3/8/2016 9:41:51 AM] OpenFile
02:31:00 DEBUG: executed DTE command File.OpenFile.
02:31:00 DEBUG: start polling for code window
02:31:00 DEBUG: polling complete
02:31:00 	Executing action [3/8/2016 9:41:52 AM] PlaceCursor
02:31:00 	Executing action [3/8/2016 9:41:52 AM] InsertText
02:31:00 	Executing action [3/8/2016 9:41:52 AM] PlaceCursor
02:31:00 	Executing action [3/8/2016 9:41:52 AM] Venus
02:31:00 	Executing action [3/8/2016 9:41:52 AM] ExecuteCommand
02:31:00 	Executing action [3/8/2016 9:41:52 AM] VerifyEditorContainsText
02:31:00 	Executing action [3/8/2016 9:41:52 AM] VerifySnippets
02:31:00 	Executing action [3/8/2016 9:41:52 AM] CloseFile
02:31:00 	Executing action [3/8/2016 9:41:52 AM] CloseFile
02:31:00  Scenario: Bug 531271: Line commit shouldn't crash doing undo/redo
02:31:00 	Executing action [3/8/2016 9:41:52 AM] OpenFile
02:31:00 DEBUG: executed DTE command File.OpenFile.
02:31:00 DEBUG: start polling for code window
02:31:00 DEBUG: polling complete
02:31:00 	Executing action [3/8/2016 9:41:52 AM] InsertText
02:31:00 	Executing action [3/8/2016 9:41:52 AM] Venus
02:31:00 	Executing action [3/8/2016 9:41:52 AM] SaveFile
02:31:00 	Executing action [3/8/2016 9:41:52 AM] CloseFile
02:31:00 	Executing action [3/8/2016 9:41:52 AM] OpenFile
02:31:00 DEBUG: executed DTE command File.OpenFile.
02:31:00 DEBUG: start polling for code window
02:31:00 DEBUG: polling complete
02:31:00 	Executing action [3/8/2016 9:41:52 AM] PlaceCursor
02:31:00 	Executing action [3/8/2016 9:41:52 AM] SendKeys
02:31:00 	Executing action [3/8/2016 9:41:53 AM] OpenFile
02:31:00 DEBUG: executed DTE command File.OpenFile.
02:31:00 DEBUG: start polling for code window
02:31:00 DEBUG: polling complete
02:31:00 	Executing action [3/8/2016 9:41:53 AM] OpenFile
02:31:00 DEBUG: executed DTE command File.OpenFile.
02:31:00 DEBUG: start polling for code window
02:31:00 DEBUG: polling complete
02:31:00 	Executing action [3/8/2016 9:41:53 AM] ExecuteCommand
02:31:00 	Executing action [3/8/2016 9:41:53 AM] ExecuteCommand
02:31:00 	Executing action [3/8/2016 9:41:53 AM] ExecuteCommand
02:31:00 	Executing action [3/8/2016 9:41:53 AM] ExecuteCommand
02:31:00 	Executing action [3/8/2016 9:41:53 AM] VerifyEditorContainsText
02:31:00 	Executing action [3/8/2016 9:41:53 AM] CloseFile
02:31:00 	Executing action [3/8/2016 9:41:53 AM] CloseFile
02:31:00  Scenario: Generate Type Into App_Code Folder
02:31:00 	Executing action [3/8/2016 9:41:54 AM] OpenFile
02:31:00 DEBUG: executed DTE command File.OpenFile.
02:31:00 DEBUG: start polling for code window
02:31:00 DEBUG: polling complete
02:31:00 	Executing action [3/8/2016 9:41:54 AM] OpenFile
02:31:00 DEBUG: executed DTE command File.OpenFile.
02:31:00 DEBUG: start polling for code window
02:31:00 DEBUG: polling complete
02:31:00 	Executing action [3/8/2016 9:41:54 AM] SetEditorText
02:31:00 	Executing action [3/8/2016 9:41:54 AM] WaitForWorkspace
02:31:00 	Executing action [3/8/2016 9:41:54 AM] WaitForFeature
02:31:00 	Executing action [3/8/2016 9:42:00 AM] WaitForFeature
02:31:00 	Executing action [3/8/2016 9:42:00 AM] PlaceCursor
02:31:00 	Executing action [3/8/2016 9:42:00 AM] WaitForFeature
02:31:00 	Executing action [3/8/2016 9:42:00 AM] VerifyCodeActions
02:31:00 	Executing action [3/8/2016 9:42:07 AM] OpenFile
02:31:00 DEBUG: executed DTE command File.OpenFile.
02:31:00 DEBUG: start polling for code window
02:31:00 DEBUG: polling complete
02:31:00 	Executing action [3/8/2016 9:42:07 AM] VerifyEditorContainsText
02:31:00 	Executing action [3/8/2016 9:42:07 AM] CloseFile
02:31:00 	Executing action [3/8/2016 9:42:07 AM] CloseFile
02:31:00 	Executing action [3/8/2016 9:42:07 AM] OpenFile
02:31:00 DEBUG: executed DTE command File.OpenFile.
02:31:00 DEBUG: start polling for code window
02:31:00 DEBUG: polling complete
02:31:00 	Executing action [3/8/2016 9:42:07 AM] SetEditorText
02:31:00 	Executing action [3/8/2016 9:42:07 AM] WaitForWorkspace
02:31:00 	Executing action [3/8/2016 9:42:07 AM] WaitForFeature
02:31:00 --------------------------------
02:31:00 System.Exception: Tao action timed out at 3/8/2016 10:22:07 AM. Timeout limit was: 00:40:00
02:31:00 ================================
```

**Comment by jmarolf** [comment]
```
Exception object: 4a6496f4
Exception type:   System.ArgumentOutOfRangeException
Message:          Specified argument was out of the range of valid values.
InnerException:   <none>
StackTrace (generated):
    SP       IP       Function
    1F19ED4C 51372566 Microsoft_VisualStudio_Text_Data_ni!Microsoft.VisualStudio.Text.SnapshotSpan..ctor(Microsoft.VisualStudio.Text.ITextSnapshot, Microsoft.VisualStudio.Text.Span)+0x33286
    1F19ED74 2F0DBBD6 Microsoft_CodeAnalysis_EditorFeatures_Text_2c410000!Microsoft.CodeAnalysis.Text.Shared.Extensions.TextSpanExtensions.ToSnapshotSpan(Microsoft.CodeAnalysis.Text.TextSpan, Microsoft.VisualStudio.Text.ITextSnapshot)+0x8e
    1F19EDBC 2F0DB288 Microsoft_CodeAnalysis_EditorFeatures_1b000000!Microsoft.CodeAnalysis.Editor.Implementation.Diagnostics.AbstractDiagnosticsTaggerProvider`1+TaggerProvider[[System.__Canon, mscorlib]].ProduceTags(Microsoft.CodeAnalysis.Editor.Tagging.TaggerContext`1<System.__Canon>, Microsoft.CodeAnalysis.Editor.DocumentSnapshotSpan)+0x3d0
    1F19EF08 2F0DAB7A Microsoft_CodeAnalysis_EditorFeatures_1b000000!Microsoft.CodeAnalysis.Editor.Implementation.Diagnostics.AbstractDiagnosticsTaggerProvider`1+TaggerProvider[[System.__Canon, mscorlib]].ProduceTagsAsync(Microsoft.CodeAnalysis.Editor.Tagging.TaggerContext`1<System.__Canon>, Microsoft.CodeAnalysis.Editor.DocumentSnapshotSpan, System.Nullable`1<Int32>)+0x3a
    1F19EF34 22D61F8D Microsoft_CodeAnalysis_EditorFeatures_1b000000!Microsoft.CodeAnalysis.Editor.Tagging.AbstractAsynchronousTaggerProvider`1+<ProduceTagsAsync>d__33[[System.__Canon, mscorlib]].MoveNext()+0x1fd
    1F19F0E8 728EDFBD mscorlib_ni!System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(System.Threading.Tasks.Task)+0x59
    1F19F0F8 728E9788 mscorlib_ni!System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(System.Threading.Tasks.Task)+0x40
    1F19F104 730D1BA3 mscorlib_ni!System.Runtime.CompilerServices.ConfiguredTaskAwaitable+ConfiguredTaskAwaiter.GetResult()+0x1b
    1F19F108 228E5603 Microsoft_CodeAnalysis_EditorFeatures_1b000000!Microsoft.CodeAnalysis.Editor.Tagging.AbstractAsynchronousTaggerProvider`1+TagSource+<RecomputeTagsAsync>d__85[[System.__Canon, mscorlib]].MoveNext()+0x24b
```

**Comment by davkean** [comment]
It looks like @heejaechang is fixing this?

**Comment by jmarolf** [comment]
fixed by #9601
